libraries/AP_InertialSensor/AP_InertialSensor_MPU6000.cpp

```diff
@@ -595,7 +595,7 @@ bool AP_InertialSensor_MPU6000::_accumulate_fast_sampling(uint8_t *samples, uint
     }
 
     if (ret) {
-        float temp = (tsum/n_samples)/340.0 + 36.53;
```

Comment:
We do for PX4 boards. But the issue here is different: tsum and n_samples are both of integer type so this will perform an integer division and so we can lose a fractional part of the division.